go-client/client/example_basic_query_test.go

```diff
@@ -22,9 +22,9 @@ func Example_basicQuery() {
 	// When starting a client connection, the client script language
 	// must match the language the server was started with,
 	// even if the client does not execute any scripts.
-	cl, err := client.NewClient(ctx, "localhost", "10000", "python")
+	cl, err := client.NewClient(ctx, test_tools.GetHost(), test_tools.GetPort(), "python")
```

Comment:
I don't think the test tools change was checked in, since I don't see the file in the changes.

Reply:
It already exists - internal/test_tools/test_tools.go. This is what the other test stuff uses already as I understand it.